ItemList/ItemList/ViewModel.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace ItemList
+{
+    /// <summary>
+    /// This is a class, that represents a ViewModel layer with all fields from View and logic how to handle requests from UI
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    public sealed class ViewModel : INotifyPropertyChanged
+    {
+        /// <summary>
+        /// Stores an instance of RelayCommand to add.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private RelayCommand addCommand;
+        /// <summary>
+        /// Stores an instance of RelayCommand to delete.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private RelayCommand deleteCommand;
+        /// <summary>
+        /// Stores instance of selected Phone.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public Phone selectedItem;
+        /// <summary>
+        /// Initializes addCommand field.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        public RelayCommand AddCommand
+        {
+            get
+            {
+                if (this.addCommand != null)
+                {
+                    return this.addCommand;
+                }
+                this.addCommand = AddNewItemToCollection();
+                return this.addCommand;
+            }
+        }
+        /// <summary>
+        /// Command adds new item to Observable Collection of phones and clean text boxes.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        private RelayCommand AddNewItemToCollection ()
```

Comment:
The name of the method shoul define what it does.
In that case your method provides, returns, creates new command. Not Adds.
It is important to give appropriate names to all members for future maintaining.
I'd give name like GetNewCommand

Reply:
I 've got what you wanted - you tried to exclude logic to a separate method as we discussed.
Lets have a talk about that.